### DISCUSSION TITLE
```with``` "setter"

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#8567] [Language Ideas] [answered]
It would be great if I could use ```with``` to set multiple properties on an existing object, similar to what I can do with object initializer syntax.

Here's an a example of what I'd like to do:
```
var P = new Person();
...

with P {
   First_Name = "John",
   Age = 21,
   Children = {
     new(){ First_Name = "James" },
   }
};

```

**Comment by 333fred** [comment, marked as answer]
Duplicate of https://github.com/dotnet/csharplang/discussions/1534.
